tomviz/TomographyReconstruction.h

```diff
@@ -0,0 +1,40 @@
+/******************************************************************************
+ 
+ This source file is part of the tomviz project.
+ 
+ Copyright Kitware, Inc.
+ 
+ This source code is released under the New BSD License, (the "License").
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ 
+ ******************************************************************************/
+
+#ifndef tomvizTomographyReconstruction_h
+#define tomvizTomographyReconstruction_h
+
+#include "pqReaction.h"
+#include "vtkImageData.h"
+
+namespace tomviz
+{
+class DataSource;
+
+class TomographyReconstruction
+{
+    
+public:
+  TomographyReconstruction();
+  ~TomographyReconstruction();

```

Comment:
If all your functions are public and static then there is no need to create a class at all in C++.  You can just put all the functions in the tomviz namespace (they don't need to be static this way) or if you want to access them with `tomviz::TomographyReconstruction::X`, then you can do a nested `namespace TomographyReconstruction` instead of a class declaration.